server.js

```diff
@@ -0,0 +1,5 @@
+const express = require('express');
+let app = express()
+app.listen(8008)
```

Comment:
good practice to have the listen at the end of the file, and indicate that it's running with a log just before